CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 # CHANGELOG
 
+### Changed
+
+- update glogger middleware to support protocol switch (e.g. from http to websocket)
+
```

Comment:
changelog file is not maintained anymore, it has been preserved for historical reason so there is no need to update it

```suggestion
```